install-panel.sh

```diff
@@ -479,6 +479,11 @@ function centos8_dep {
 #################################
 
 function ubuntu_universedep {
+  #Probably should change this, this is more of a bandaid fix for this 
+  #This function running before software-properties-common is installed
+  apt update -y
```

Comment:
Is the `apt update -y` required here?

Reply:
Not sure, never tested.
I just put it in there for good measure

Reply:
Looks like it is needed
You have made your own function to update and upgrade the debian package manager

though at the start of the function "perform_install" the "ubuntu_universedep" is called before the first time "apt_update" function is called.

So to get it installed my way, it has to have the apt update, as some people install this on old systems that haven't been updated in a long time. So I suggest leaving it in to keep my suggested fix for good measure

Reply:
👍